.github/CONTRIBUTING.md

```diff
@@ -86,66 +86,32 @@ to work on the fork is to set it as a remote of the Consul project:
 By following these steps you can push to your fork to create a PR, but the code on disk still
 lives in the spot where the go cli tools are expecting to find it.
 
->Note: If you make any changes to the code, run `make format` to automatically format the code according to Go standards.
+>Note: If you make any changes to the code, run `gofmt -s -w` to automatically format the code according to Go standards.
 
 ## Testing
 
-### During Development: Run Relevant Test(s)
-
 During development, it may be more convenient to check your work-in-progress by running only the tests which you expect to be affected by your changes, as the full test suite can take several minutes to execute. [Go's built-in test tool](https://golang.org/pkg/cmd/go/internal/test/) allows specifying a list of packages to test and the `-run` option to only include test names matching a regular expression.
+The `go test -short` flag can also be used to skip slower tests.
 
 Examples (run from the repository root):
 - `go test -v ./connect` will run all tests in the connect package (see `./connect` folder)
 - `go test -v -run TestRetryJoin ./command/agent` will run all tests in the agent package (see `./command/agent` folder) with name substring `TestRetryJoin`
 
-### Before Submitting Changes: Run All Tests
-
-Before submitting changes, run **all** tests locally by typing `make test`. 
-The test suite may fail if over-parallelized, so if you are seeing stochastic 
-failures try `GOTEST_FLAGS="-p 2 -parallel 2" make test`. 
-
-Certain testing patterns such as creating a test `Client` in the `api` pkg 
-or a `TestAgent` followed by a session can lead to flaky tests. More generally, 
-any tests with components that rely on readiness of other components are often
-flaky. 
-
-Our makefile has some tooling built in to help validate the stability of single 
-or package-wide tests. By running the `test-flake` goal we spin up a local docker 
-container that mirrors a CPU constrained version of our CI environment. Here we can 
-surface uncommon failures that are typically hard to reproduce by re-running 
-tests multiple times. 
-
-The makefile goal accepts the following variables as arguments:
-
-* **FLAKE_PKG** Target package (required)
-
-* **FLAKE_TEST** Target test
-
-* **FLAKE_CPUS** Amount of CPU resources for container
-
-* **FLAKE_N** Number of times to run tests
-
-Examples:
-`make test-flake FLAKE_PKG=connect/proxy`
-`make test-flake FLAKE_PKG=connect/proxy FLAKE_TEST=TestUpstreamListener`
-`make test-flake FLAKE_PKG=connect/proxy FLAKE_TEST=TestUpstreamListener FLAKE_CPUS=0.15 FLAKE_N=30`
+When a pull request is opened CI will run all tests and lint to verify the change.
```

Comment:
Is this true? I think I've seen some PRs require approval from a `member` to run some CI tests (maybe to prevent malicious test runs)

Reply:
Good question. Github has added a prompt for first time contributors that requires a member to click it. But I believe that only applies to github actions. I think I've seen CircleCI checks run even without that button being clicked. It's possible I don't fully understand how that new button works, but either way, one way or another we will use CI to verify the change.

There are some cases where CircleCI just doesn't seem to want to run anything (ex: #10701), but I think that is due to a bug. Normally CI should run on all PRs.